src/d3d9/d3d9_common_texture.cpp

```diff
@@ -26,6 +29,10 @@ namespace dxvk {
       for (uint32_t i = 0; i < subresources; i++) {
         SetNeedsUpload(i, true);
       }
+      if (pSharedHandle) {
+        Logger::warn("D3D9CommonTexture: Shared resources must use D3DPOOL_DEFAULT");
+        pSharedHandle = nullptr;
```

Comment:
Shouldn't we fail texture creation instead?

Reply:
Yeah, looks like we should.